doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1832,18 +1833,12 @@ Response code:
          {
             "type": "Room",
             "id": "DC_S1-D41",
-            "temperature": {
```

Comment:
This (and the next one) correspond to a very recent change in the .apib. It should desapear after merging develop into hardening/csub_documentation branch.